frontend/www/js/omegaup/components/problem/Details.vue

```diff
@@ -405,12 +406,20 @@ export default class ProblemDetails extends Vue {
     );
   }
 
-  get parsedQualityHistogram(): number[] {
+  get parsedQualityHistogram(): null | number[] {
+    const qualityHistogram = this.histogram?.qualityHistogram;
+    if (!qualityHistogram) {
+      return null;
+    }
     return JSON.parse(this.histogram?.qualityHistogram ?? '');
```

Comment:
```suggestion
    return JSON.parse(qualityHistogram);
```

por consistencia con L422

Reply:
Listo